package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "open-msupply",
   "//": "Main version for the app, should be in semantic version format (any release candidate or test build should be separated by '-' i.e. 1.1.1-rc1 or 1.1.1-test",
-  "version": "2.3.00",
+  "version": "2.3.00-RC5",
```

Comment:
Should we leave this as 2.3.00? Or is develop now considered 2.4 👀

Reply:
Can change now or leave. Usually the first person who does a migration changes but it's all goodd